apps/prairielearn/src/sync/fromDisk/questions.ts

```diff
@@ -72,10 +69,8 @@ export async function sync(courseId, courseData) {
   const params = [questionParams, courseId];
 
   perf.start('sproc:sync_questions');
-  const result = await sqldb.callOneRowAsync('sync_questions', params);
+  const result = await sqldb.callRow('sync_questions', params, z.record(z.string(), IdSchema));
```

Comment:
Same comment as above.

Reply:
Done in 6934d26; I got rid of all the other separate `params` objects/arrays in that commit as well.